components/renku_data_services/notebooks/apispec.py

```diff
@@ -259,10 +259,10 @@ class SessionLogsResponse(RootModel[Optional[Dict[str, str]]]):
 
 
 class SessionCloudStoragePost(BaseAPISpec):
-    configuration: Dict[str, Any]
-    readonly: bool = True
-    source_path: str
-    target_path: str
+    configuration: Optional[Dict[str, Any]] = None
+    readonly: Optional[bool] = None
+    source_path: Optional[str] = None
+    target_path: Optional[str] = None
```

Comment:
nitpick ([PEP604](https://peps.python.org/pep-0604/)):
```suggestion
    configuration: dict[str, Any] | None = None
    readonly: bool | None = None
    source_path: str | None = None
    target_path: str | None = None
```

Reply:
This is auto generated code. And it has this old style of typing everywhere - not just in notebooks. There is a `--use-union-operator` option that I can add. But I will do that in a followup PR on main. Rebasing this and the feature branch against main is already a pain. This will make it harder.

Reply:
Oh, this should be dismissed then.